inst/WORDLIST

```diff
@@ -311,6 +311,7 @@ modularized
 msec
 nd
 occds
+onboarding
```

Comment:
I think this is not necessary. You can check by calling `spelling::update_wordlist()`.